### DISCUSSION TITLE
Consulta sobre puntos TP3

### DISCUSSION THREAD

**Discussion by alengeor** [INGCOM-UNRN-P1/cursada-2023#89] [Preguntas y respuestas] [answered]
Estimados, como andan?
Consulta, esta vez mas por modo de entrega que por código. ¿Hay que hacer un unico código/archivo por punto? Es decir, uno para los puntos 1.1, 1.2, etc y otro para los puntos 2.1, 2.2, etc. En el punto dos me parece que está claro, pero del punto 1 no dice nada:
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/110181852/33188b81-2e2c-475e-ae1b-c0c0638a4888)
Que entienden uds? 🤔

**Comment by martinvilu** [comment, marked as answer]
El práctico son dos archivos, uno por cada punto.
